tests/test_utils/amazon_system_helpers.py

```diff
@@ -82,6 +89,31 @@ def delete_aws_s3_bucket(cls, name: str) -> None:
         cmd = ["aws", "s3api", "delete-bucket", "--bucket", name]
         cls.execute_with_ctx(cmd)
 
+    @classmethod
+    def create_aws_s3_key(cls, bucket_name: str, key_name: str, body: Optional[str] = None) -> None:
+        """
+        Puts an object into the aws bucket.
+
+        :param bucket_name: name of the bucket
+        :param key_name: name of the key to add
+        :param body: optional content of the key to add
+        """
+        cmd = ["aws", "s3api", "put-object", "--bucket", bucket_name, "--key", key_name]
```

Comment:
Ummm. We have boto installed, let's use that instead of shelling out

Reply:
True, but my thought is that we should not system-test our prod code with prod code? I mean theoretically if we change our code to use a different lib or so we also might need to change our test code.

Reply:
Also some function we don't have in our hooks or operators. We then would need to mix hook, operator and boto3 calls. I think it would get confusing / different abstraction layers.

Reply:
Of course, terraform would be better to create resources but since we have `awscli` installed anyway, I think it is not bad to start like this and switch to terraform once we have it.